packages/oui-numeric/src/numeric.component.js

```diff
@@ -0,0 +1,18 @@
+import template from './numeric.html'
+import controller from './numeric.controller'
+
+export default {
+  template,
+  controller,
+  bindings: {
+    id: '@',
+    name: '@?',
+    text: '@',
+    unit: '@?',
+    model: '=',
+    min: '<?',
```

Comment:
I would have used `@?` instead of `<` for `min` and `max`, what do you think ?

I think user will more use our component in this way: `<oui-numeric min="2" max="100"></oui-numeric>` than `<oui-numeric min="$ctrl.min" max="$ctrl.max"></oui-numeric>`